test/libsolidity/util/BytesUtils.cpp

```diff
@@ -96,6 +96,25 @@ bytes BytesUtils::convertNumber(string const& _literal)
 	}
 }
 
+bytes BytesUtils::convertFixedPoint(string const& _literal, size_t *_fixedPointN)
+{
+	solAssert(_fixedPointN != nullptr, "");
+	solAssert(_literal.find('.') != string::npos, "");
+	string v_integer{_literal};
+	string v_fraction;
+	v_integer = _literal.substr(0, _literal.find('.'));
+	v_fraction = {_literal.substr(_literal.find('.') + 1)};
+	try
+	{
+		*_fixedPointN = v_fraction.length();
+		return util::fromHex(util::toHex(u256{v_integer + v_fraction}));
```

Comment:
There should be other functions to convert a `u256` to bytes that does not go through hex